linode/manager
# refactor: [M3-6316, M3-6317] MUI v5 Migration - Components > DateTimeDisplay, DebouncedSearchTextField

## Description 📝

**What does this PR do?**

- Ran the tss-react codemod on DebouncedSearchTextField, but then replaced with `styled` since it was such a small change
- No styles to run codemod on DateTimeDisplay
- Modernized both components by using named exports and better prop interface names

## Preview 📷
DebouncedSearchTextField:

![image](https://user-images.githubusercontent.com/114685994/232524644-6dc4d98b-8826-4ace-9fb3-420b416924dc.png)

DateTimeDisplay:

![Screenshot 2023-04-17 at 7 58 20 AM](https://user-images.githubusercontent.com/114685994/232525105-b739194c-8146-41dd-ab8a-ee1cae60d8fd.jpg)

## How to test 🧪

**What are the steps to reproduce the issue or verify the changes?**
- Check that there have been no changes visually or in functionality for the DebouncedSearchTextFields throughout the app
   - e.g. http://localhost:3000/stackscripts/community
- Check that there have been no visual changes for the DateTimeDisplay throughout the app
   - e.g. http://localhost:3000/events
- Check that Storybook is still rendering correctly: http://localhost:6006/?path=/docs/elements-typography-date-time-display--docs

**How do I run relevant unit or e2e tests?**

``` yarn test DateTimeDisplay ```
